packages/labs/nextjs/README.md

```diff
@@ -0,0 +1,39 @@
+# @lit-labs/ssr-react
+
+A package for integrating Lit SSR with Next.js.
```

Comment:
micro nit: Kind of like how you should start a classes jsdoc with "A class that ...", everything on npm is a package, so you can say "Integrates Lit SSR..."

```suggestion
Integrates Lit SSR with Next.js to enable deep server rendering of Lit components.
```